BepInEx.Core/Utility.cs

```diff
@@ -221,6 +221,55 @@ public static bool TryResolveDllAssembly<T>(AssemblyName assemblyName,
         return false;
     }
 
+    /// <summary>
+    ///     Try to resolve the symbols data from an assembly DLL.
+    /// </summary>
+    /// <param name="dllPath">File path of the DLL assembly.</param>
+    /// <param name="symbolsData">The symbols of the DLL assembly.</param>
+    /// <returns>True, if the symbols were found. Otherwise, false.</returns>
+    public static bool TryResolveDllSymbols(string dllPath, out byte[] symbolsData)
+    {
+        symbolsData = null;
+        
+        var subDirectory = Path.GetDirectoryName(dllPath);
+        var filename = Path.GetFileNameWithoutExtension(dllPath);
+
+        var potentialSymbolsFileNames = new[]
+        {
+            $"{filename}.mdb",
```

Comment:
`.mdb` is actually a suffix to the existing full file name like so `AssemblyName.dll.mdb`